src/util/sss_krb5.c

```diff
@@ -973,12 +980,6 @@ krb5_error_code sss_extract_pac(krb5_context ctx,
         goto done;
     }
 
-    ret = unsetenv("_SSS_LOOPS");
```

Comment:
Was it here by mistake?

Reply:
It was just on an odd place, I moved it to be nearer to the actual send.

Reply:
But previously it wasn't restored.